src/main/java/com/minecolonies/coremod/colony/CitizenData.java

```diff
@@ -21,8 +30,13 @@
 /**
  * Extra data for Citizens.
  */
+@SuppressWarnings(Suppression.BIG_CLASS)
 public class CitizenData
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') class "CitizenData" has 51 methods, which is greater than the 35 authorized. Split it into smaller classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1448)